### ISSUE TITLE
Can't compile code gml_GlobalScript_scr_gamestart in Deltarune 1.07 with UTMT 0.4.0.1

### ISSUE THREAD

**Issue by SheepYhangCN** [krzys-h/UndertaleModTool#625]
when i tried to enable debug(in code gml_GlobalScript_scr_gamestart line243), it can't compile
UTMT version:0.4.0.1
Deltarune version:1.07(Steam version and Chinese translate version are same)

screenshot:
![image](https://user-images.githubusercontent.com/44539204/136379120-89d0bcb0-1a1a-4c38-81f0-b00e7e1a4b2d.png)

**Comment by Romualdo666** [comment]
how can I do any of these?: "edit assembly or use a dummy GameMaker project."

**Comment by Jacky720** [comment]
To edit assembly, select the "Disassembly" tab in the tool. It's lower-level than decompiled code and less prone to breaking.

This document explains what different commands do: https://docs.google.com/document/d/132mmSdLm2iQJIFBPQXcQsc0d7lBJZinkA_gZIVVLaqo/edit?usp=drivesdk

For example, the lines that disable global.debug are:
```
pushi.e 0
pop.v.i global.debug
```
And one would want to change the 0 to a 1.

**Comment by Romualdo666** [comment]
oh thanks, but thats only for active the debug mode or I can modify Global Scripts too?
becouse I'm making a mod and I really need to modify Global scripts and compile them

**Comment by Jacky720** [comment]
Yes, this method also works on scripts. Just don't mess with the first or last couple of lines.

**Comment by Romualdo666** [comment]
Thanks

**Comment by Romualdo666** [comment]
sorry one more question: then can I add a new key item at gmlGlobalScript_scr_keyiteminfo?

**Comment by Jacky720** [comment]
In the wise words of the Underminers FAQ channel:

Q: Can I do \*insert something here* to Undertale or Deltarune?
A: Yes, you can do anything, so long as you are determined enough.

**Comment by Romualdo666** [comment]
well thanks, again.

**Comment by Jacky720** [comment]
With functional script compilation in addition to the workarounds above, this issue should be closed.
